web/styles/settings.css

```diff
@@ -1447,14 +1452,12 @@ label.preferences-radio-choice-label {
             text-align: center;
             font-size: 1.1em;
             line-height: 1;
-            margin: 15px;
             text-transform: uppercase;
         }
 
         .exit {
             font-weight: 600;
             position: absolute;
-            top: 10px;
```

Comment:
This isn't necessary anymore because the contents are vertically centered with flex. The `position:absolute` and `right` are still necessary to place the `x` on the right